File: programacao/c/respostas/selecao/43.c
#include <stdio.h>

int main(void) {
    float hdl, ldl, r;

    printf("HDL (mg/dL): ");
    scanf("%f", &hdl);
    printf("LDL (mg/dL): ");
    scanf("%f", &ldl);

    r = hdl / ldl;

    printf("Razao HDL/LDL: %.1f\n", r);

    if (r >= 0.4) {
        printf("Risco cardiovascular baixo\n");
    } else if (r > 0.3) {
        printf("Risco cardiovascular moderado\n");
    } else {
        printf("Risco cardiovascular elevado\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/43.py
hdl = float(input("HDL (mg/dL): "))
ldl = float(input("LDL (mg/dL): "))

r = hdl / ldl

print(f"Razao HDL/LDL: {r:.1f}")

if r >= 0.4:
    print("Risco cardiovascular baixo")
elif r > 0.3:
    print("Risco cardiovascular moderado")
else:
    print("Risco cardiovascular elevado")
